lib/Transforms/HandshakeOptimizeBitwidths.cpp

```diff
@@ -670,6 +675,7 @@ struct HandshakeMuxSelect : public OpRewritePattern<handshake::MuxOp> {
     auto newMuxOp = rewriter.create<handshake::MuxOp>(
         muxOp.getLoc(), muxOp->getResultTypes(), newOperands,
         muxOp->getAttrs());
+    inheritBB(muxOp, newMuxOp);
```

Comment:
Unnecessary, the new op inherits the attributes from the old one. We call `replaceOp` on the namer to update the namer's internal state, not the attribute itself which has already been forwarded.